cmake/cross-toolchain-mingw32.cmake

```diff
@@ -0,0 +1,16 @@
+# Target operating system and architecture
```

Comment:
What is this for? Are you adding a mingw CI build?

Reply:
I want to add MinGW to the crunch Azure CI.

It happens that MinGW 32-bit and MinGW 64-bit brought me different problems to fix, building one successfully was not enough to know the other build would build as successfully.